modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-ack.adoc

```diff
@@ -0,0 +1,153 @@
+= Anypoint MQ ACK and NACK Operations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+== Acknowledgment Configurations
+
+The Anypoint MQ connector provides three acknowledgment configurations: Immediate, Automatic, and Manual.
+
+=== Immediate Acknowledgment
+
+To acknowledge messages as your app consumes them, before processing them, set the `acknowledgementMode` to `IMMEDIATE` either in the Subscriber or Consume operations.
+
+Acknowledging the message automatically when it's received means that the message isn't redelivered if an error occurs during message processing. To handle errors without losing messages, use a dead letter queue.
+
+If an error occurs while trying to acknowledge the message using `IMMEDIATE` mode, an `ANYPOINT-MQ:ACKING` error is thrown.
```

Comment:
"If an error occurs while trying to..." contains an unclear reference to what is "trying to...." 
Because this sentence has a context provided by the previous sentences, can we use...:

     "If an error occurs during message processing using 'IMMEDIATE',..."